scripts/check-evidence-ids.js

```
@@ -0,0 +1,52 @@
+const fs = require('fs');
+const path = require('path');
+const glob = require('glob');
+
+// Since we are in a script, we might need to install glob or use a simpler approach.
+// For simplicity in this CI script, let's assume we can use basic fs or that glob is available.
+// If glob is not available, we can use a recursive readdir function.
+
+function getAllFiles(dirPath, arrayOfFiles) {
+  files = fs.readdirSync(dirPath)
```

Biome (2.3.13)

[error] Can't assign files because it's a constant.

This is where the variable is defined as constant.

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)